src/main/java/net/pms/util/BitRateMode.java

```diff
@@ -0,0 +1,52 @@
+package net.pms.util;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import java.util.Locale;
+
+/**
+ * This {@code enum} represents the possible bit rate "modes".
+ */
+public enum BitRateMode {
+
+	/** Constant bit rate mode*/
+	CBR,
+
+	/** Variable bit rate mode */
+	VBR;
+
+	@Override
+	public String toString() {
+		switch (this) {
+			case CBR:
+				return "Constant";
```

Comment:
I know that you like verbose, and i could understand that it can be more meaningful.
But as you know, i like more CBR or VBR, it's more log readable to be. Too much text / verbose kill the informations; it was just a share of my point of view only, so i'll not bargain for a change.